framework/16-inch/amd-ai-300-series/nvidia/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, ... }:
+
+{
+  imports = [
+    ../.
+    ../../../../common/gpu/nvidia/blackwell
+    ../../../../common/gpu/nvidia/prime.nix
+  ];
+
+  # Explicitly set nvidia as video driver to override modesetting from AMD module
+  services.xserver.videoDrivers = [ "nvidia" ];
+
+  hardware.nvidia = {
+    # Hybrid graphics with PRIME offload for better battery life
+    # AMD iGPU by default, NVIDIA dGPU on demand via nvidia-offload command
+    prime = {
+      # Bus IDs can be found with `lspci | grep -E "VGA|3D"`
```

```suggestion
      # Bus IDs can be found with `lspci | grep -E "VGA|3D|Display"`
```

I need to grep for Display to show the internal GPU

```
> lspci | grep -E "VGA|3D|Display"
c2:00.0 VGA compatible controller: NVIDIA Corporation GB206M [GeForce RTX 5070 Max-Q / Mobile] (rev a1)
c3:00.0 Display controller: Advanced Micro Devices, Inc. [AMD/ATI] Strix [Radeon 880M / 890M] (rev c1)
```